frontend/homework/9/src/index.scss

```diff
@@ -0,0 +1,10 @@
+body{
```

Comment:
All the constants, font weights, sizes and the colors must be stored in the constant file